### ISSUE TITLE
Favorites Import Nothing!

### ISSUE THREAD

**Issue by seniorm0ment** [osmandapp/OsmAnd#17049]
I had to reinstall OSM, so I exported everything and then uninstalled and reinstalled. I then imported from the file, clicked replace all and it shows everything including the favorites gpx files. It told me it needs to restart the app, I do that and it it reopens and there's no favorites at all!

I also tried an old backup and it didn't import anything either

**Comment by seniorm0ment** [comment]
If I try reimporting a second time, it says that the files already exist in osmand, and I even tried overwriting those with the imports and it still did not resolve.

**Comment by scaidermern** [comment]
Does this mean My places / Favorites is completely empty? Or are your favorites maybe listed there, but are disabled?

**Comment by Zirochkabila** [comment]
OsmAnd~ 4.5.0#35841m, Released: 2023-04-24
@seniorm0ment It was tested both from the file containing all the data and from the file containing only the Favorites - all the data was imported
Can you check my files?

[Archive 4.zip](https://github.com/osmandapp/OsmAnd/files/11320341/Archive.4.zip)

**Comment by seniorm0ment** [comment]
Very strange, so I just tried yours and it worked. 
However mine was not separate files, it was just the main .osf export.

That being said, I just tried once more with mine and it actually worked this time. Not sure what went wrong, because I tried with the backup files I had, as well as the new export. Also I had tried reinstalling like 3 times.

Feel free to close this issue, unless you think it could be a bug
